### PR TITLE
update Dockerfile for ovn2.11

### PR DESCRIPTION
**Pull request by pecameron**
Update ovn rpms from openvswitch2.11-ovn to ovn2.11

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@dcbw @squeed @wking 
Got an email "Failed OCP build of openshift/ose-ovn-kubernetes:v4.4.0" that looks like a renaming problem for the ovn rpms. PTAL

**Comment by pecameron** [comment]
@dcbw PTAL

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +9,7 @@
 # are built in this Dockerfile and included in the image (instead of the rpm)
 #
 
+
```
Comment (dcbw) [inline Dockerfile]: @pecameron unecessary whitespace addition?

**Comment by dcbw** [comment]
@pecameron can we merge #48 into this one? We want to use openvswitch2.12 and ovn2.11

File: Dockerfile (hunk 1 above)
Comment (pecameron) [reply Dockerfile]: As suggested, forced tests to run again. I'll delete it.

**Comment by pecameron** [comment]
PR 48 replaces this.

**Pull request closed without merging**